test/verify/test_gemm_copy.cpp

```diff
@@ -16,9 +17,8 @@ struct test_gemm_copy : verify_program<test_gemm_copy>
         auto pa = mm->add_parameter("a", sa);
         auto pb = mm->add_parameter("b", sb);
         auto pc = mm->add_parameter("c", sc);
```

Comment:
You can change sc to {migraphx::shape::float_type, {1, 8}} to have one of the above lines covered for code coverage.

Reply:
Done